### PR TITLE
Issue Title

### LINKED ISSUE #10: Issue Title
idk

### PR DESCRIPTION
**Pull request by andreilg**
Fixes #10

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Acme Corporation&trade; is the world's leading manufacturer of digital shapes. F
 
 > We shape the world. You live in it.
 
-We provide :smile: to hundreds of happy customers.
+We provide meme :smile: to hundreds of happy customers.
```
Comment (andreilg) [inline content/about.md:12]: Good job

File: content/about.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Acme Corporation&trade; is the world's leading manufacturer of digital shapes. F
...
 
 ## Manufacturing
```
Comment (andreilg) [inline content/about.md:14]: idk

**Review by andreilg — COMMENTED** [review]
ik

File: content/about.md (hunk 2 above)
Comment (andreilg) [inline content/about.md:14]: not good

Comment (wkenned1) [reply content/about.md:14]: meme

**Pull request merged**